choreographer/src/main/resources/application.properties

```diff
@@ -35,6 +35,14 @@ log_all_request_and_response=false
 sr_address=127.0.0.1
 sr_port=8443
 
+# If true, using notify URL address with self-addressing IPv4, IPv6 and no-type addresses are allowed.
+# In case of self-addressing addresses the IP packets cannot be directed from one device to another.
+allow_self_addressing=true
+
+# If true, using notify URL address with non-routable IPv4 and IPv6 addresses are allowed.
+# In case of non-routable addresses the IP packets cannot be directed from one network to another.
+allow_non_routable_addressing=true
+
```

Comment:
I would try to avoid the usage these props in Choreographer and let all the responsibility to SR. I belive you put here these because of the executors address detection (where I've forgotten about), but the `use_network_address_detector` prop is flase in default, so without that, these have no effect.

Maybe we should use only the `use_network_address_detector` and the `filter_proxy_addresses` props here and refactor the `detect(final HttpServletRequest servletRequest)` in NetworkAddressDetector to `detect(final HttpServletRequest servletRequest, final boolean verify)` and skip the verification in Choreographer. (SR will do it)

Reply:
No, I put there because of notify uris.